internal/builders/container/README.md

```diff
@@ -85,6 +86,7 @@ jobs:
       packages: write
     outputs:
       image: ${{ steps.image.outputs.image }}
+      digest: ${{ steps.build.outputs.digest }}
```

Comment:
I added the digest to the docs in #591 so I think this will be unnecessary if you merge with main.

Reply:
thanks for the hint you're right, we can drop this one